swap/src/protocol/bob.rs

```diff
@@ -149,33 +149,33 @@ impl Behaviour {
     /// Sends a message to Alice to get current amounts based on `btc`.
     pub fn request_amounts(&mut self, alice: PeerId, btc: u64) {
         let btc = ::bitcoin::Amount::from_sat(btc);
-        let _id = self.amounts.request_amounts(alice.clone(), btc);
+        let _id = self.amounts.request_amounts(alice, btc);
         info!("Requesting amounts from: {}", alice);
     }
 
     /// Sends Bob's first message to Alice.
     pub fn send_message0(&mut self, alice: PeerId, msg: bob::Message0) {
         self.message0.send(alice, msg);
-        debug!("Sent Message0");
+        info!("Sent Message0");
```

Comment:
Any reason to upgrade this? I think `debug` is appropriate, a user does not really care about that.

Reply:
I will remove this commit. The amounts message is `info` for some reason, the rest are debug.